utils/node-fuzzer/src/runtime/account.rs

```diff
@@ -0,0 +1,62 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use gear_common::Origin;
+use gear_runtime::Runtime;
+use pallet_gear::BlockGasLimitOf;
+use runtime_primitives::{AccountId, AccountPublic, Balance};
+use sp_consensus_babe::AuthorityId as BabeId;
+use sp_core::{sr25519::Public, Pair, Public as TPublic};
+use sp_finality_grandpa::AuthorityId as GrandpaId;
+use sp_runtime::{app_crypto::UncheckedFrom, traits::IdentifyAccount};
+
+pub const ALICE: u64 = 1_000_001;
```

Comment:
Seems that it's not Alice

Reply:
is there a known `u64` constant for Alice?

Reply:
nope, Alice isn't u64 generated

Reply:
intoduced